Signup and Login Pages

harris-ali1 opened the pull request:
Signup and Login pages have been created.
Social media links have been made functional and alignment issues with the footer fixed

harris-ali1 committed:
Signup and Login Pages

Signup and Login pages have been created.
Social media links have been made functional and alignment issues with the footer fixed

harris-ali1 committed:
admin panel and login and sign up pages

Zentro commented on pages/signup.vue:
Don't use css classes here, only tailwind styling.

Zentro commented on pages/login.vue:
Same here, tailwind only.

harris-ali1 committed:
Tailwind applied to login and signup

As per instruction's the login and signup pages were converted to Tailwind to fix MR issues.
